packages/@okta/vuepress-site/docs/guides/revoke-tokens/removeusersess/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Remove a User session
+---
+
+Separate from access and refresh tokens, there is also the Okta session cookie that provides access to your Okta organization and applications. For a more complete explanation of Okta User sessions, see the [Sessions API documentation](/docs/reference/api/sessions/). You can revoke Okta sessions in one of two ways:
+
+* Close a specific session using the Sessions API
+* Revoke all sessions for a given user using the Users API
+
+> **Note:** Removing all User sessions can optionally also remove all related access and refresh tokens.
```

Comment:
I don't think this note is clear on how removing all User sessions can remove all related access and refresh tokens. The developer should answer: "What needs to be added to the removal request in order to remove the related access and tokens?"